doc/user/formats.rst

```diff
@@ -130,3 +130,69 @@ fractional_ranges
         be masked if :math:`[\frac{i-0.5}{nchans}, \frac{i+0.5}{nchans}]`
         overlaps any of the ranges. Note that this means that channel 0
         is centred at 0.0 but extends below it.
+
+Primary beam
+------------
+The file contains the information in a compact Fourier-transform ("aperture
+plane") form. It comprises a 3-dimensional array J of Jones matrices. Another
+three arrays :math:`\nu`, :math:`y` and :math:`x` indicate the frequency (in
+Hz) and spatial position (in metres) of each sample along the axes.
+
+To determine the response at a sampled frequency :math:`\nu_f` and some
+direction, turn the direction into direction cosines :math:`l` and :math:`m`
+relative to the pointing centre (as defined in
+:class:`katsdpmodels.primary_beam.AltAzFrame`). Then the Jones
+matrix for the response is
+
+.. math:: \frac{1}{|x|\cdot|y|}
+          \sum_{j,k} e^{-2\pi i (x_j l + y_k m)\nu_f/c} J_{f,k,j},
```

Comment:
I think you still need to comment that i=sqrt(-1) here. I must add that I am more used to using j=sqrt(-1) than i from when I went to university. Not sure in which field which one is preferred, but in all my Fourier classes it was always j.

Reply:
From which we deduce that you studied engineering :-) But surely whether you see 2πi or 2πj in an exponent in the context of a Fourier transform you're going to know what's meant (now that I've removed the use of `i` to mean something else)?

Reply:
Yes. It wont hurt to say i=sqrt(-1) explicitly or explicitly say x is indexed by j, etc. I think the |x| notation may also be ambiguous and could make the reader think of sqrt(sum_i |x_i|**2). But then again I don't really read documentation so maybe it is futile anyways.

Reply:
The |x| I did clarify - I agree it could be misread as a vector norm.